test_exercise1.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+import unittest
+from exercise1 import print_greeting
+
+class GreetUserTest(unittest.TestCase):
+    """Test for function print_greeting"""
+
+    def test_greeting(self):
+        """Test if greeting returns user's name with a greeting"""
+
+        name = "amy"
```

Comment:
Perfect! One recommendation that i made in the first PR in this repo is separating the phases of the tests with blank lines.